blueprints/app/files/package.json

```diff
@@ -12,14 +12,14 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
-    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
+    "lint": "npm-run-all -n --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all -n --aggregate-output --continue-on-error --parallel 'lint:*:fix'",
```

Comment:
Is there a long form for `-n`? I'd much prefer to have the long form names for commited scripts like this.

Reply:
Why do we move to `'` for the `lint:fix` script but not `lint`?

Reply:
> Is there a long form for `-n`? I'd much prefer to have the long form names for commited scripts like this.
Fair point. There is a long form. `--print-name` Ill make a revision to include that. 

> Why do we move to ' for the lint:fix script but not lint?
I'll address this in my next revision.